src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -469,23 +435,23 @@ public void selectQuestionNumber(int qnNumber, int newQnNumber){
      * {@code False} if not.
      */
     public boolean verifyEditSessionBoxIsEnabled() {
-        boolean isEditSessionEnabled = fsSaveLink.isDisplayed() && timezoneDropDown.isEnabled();
-        isEditSessionEnabled &= neverSessionVisibleTimeButton.isEnabled();
-        isEditSessionEnabled &= defaultSessionVisibleTimeButton.isEnabled();
-        isEditSessionEnabled &= customSessionVisibleTimeButton.isEnabled();
+        boolean isEditSessionEnabled = fsSaveLink.isDisplayed() && timezoneDropDown.isEnabled()
+                                       && neverSessionVisibleTimeButton.isEnabled()
+                                       && defaultSessionVisibleTimeButton.isEnabled()
+                                       && customSessionVisibleTimeButton.isEnabled()
+                                       // ...&&...
```

Comment:
Line separator.
The previous was separated into logical parts, but after joining them, leaving a blank line will be odd.
An alternative would be to reassign the variable.
````
isEnabled = etc && etc
                && etc;
isEnabled = isEnabled && etc
                      && etc;
````
(or to erase the commit and not bother short-circuiting the assignment.)